tests/opentelemetry/flask_app.py

```diff
@@ -21,9 +21,9 @@ def index():
 
 @app.route("/otel")
 def otel():
-    oteltracer = opentelemetry.trace.get_tracer(__name__)
-    with oteltracer.start_as_current_span("otel-flask-manual-span"):
-        return "otel", 200
+    with ddtrace.tracer.trace(name="internal", resource="otel-flask-manual-span"):
```

Comment:
Is it not confusing that we're generating a ddtrace span for this otel endpoint? I'm worried this change causes us to no longer test what were intending to? Doesn't this make us "test_distributed_trace_with_flask_app" with ddtrace mechanism, rather than otel?

Reply:
An alternative is to remove the `ddtrace-run` test. Here we can just test with the otel autoinstrumentation. We already have sufficient test coverage for parenting otel and ddtrace spans ex: https://github.com/DataDog/dd-trace-py/blob/5408c2cdc1205c5932b0157b3c77e98e54cc2806/tests/opentelemetry/test_context.py#L12-#L68.

Reply:
I don't think it's worth the time investment to debug riot

Reply:
@ZStriker19 I looked into this further. The issue was not riot. The `OTEL_PYTHON_CONTEXT` enviornment variable was not being set. I fixed this issue here: https://github.com/DataDog/dd-trace-py/pull/8017/commits/17f4dff503bf9b2391134e20d5a08e9a7fc7fd7b.

I reverted the `ddtrace.tracer.trace(....)` change

Reply:
Ah good job figuring it out! Glad it wasn't a riot bug.